### PR TITLE
use stady materials service

### PR DESCRIPTION
**Pull request by davgur**

### CODE REVIEW

File: src/apps/VirtualApp/components/HomerLimud.js [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+import { List, ListItem, MenuItem, TextField } from '@material-ui/core';
+import { getLanguage } from '../../../i18n/i18n';
+import Accordion from '@material-ui/core/Accordion';
+import AccordionSummary from '@material-ui/core/AccordionSummary';
+import AccordionDetails from '@material-ui/core/AccordionDetails';
+import Typography from '@material-ui/core/Typography';
+
+const fetchMessages = async () => {
+  try {
+    const res = await fetch('https://stmat.kli.one/api/units', { method: 'GET' });
```
Comment (amnonbb) [inline src/apps/VirtualApp/components/HomerLimud.js:12]: Please put url into env

**Review by amnonbb — CHANGES_REQUESTED** [review]

File: src/apps/VirtualApp/components/HomerLimud.js (hunk 1 above)
Comment (davgur) [reply src/apps/VirtualApp/components/HomerLimud.js]: done

File: src/apps/VirtualApp/components/HomerLimud.js [hunk 2]
```diff
@@ -9,7 +9,7 @@ import Typography from '@material-ui/core/Typography';
 
 const fetchMessages = async () => {
   try {
-    const res = await fetch('https://stmat.kli.one/api/units', { method: 'GET' });
+    const res = await fetch(FETCH_STUDY_MATERIALS, { method: 'GET' });
```
Comment (amnonbb) [inline src/apps/VirtualApp/components/HomerLimud.js:12]: you forgot export ;)

**Review by amnonbb — CHANGES_REQUESTED** [review]

File: src/apps/VirtualApp/components/HomerLimud.js (hunk 2 above)
Comment (davgur) [reply src/apps/VirtualApp/components/HomerLimud.js]: you mean import? fixed

Comment (davgur) [reply src/apps/VirtualApp/components/HomerLimud.js]: for .env rename to REACT_APP_STUDY_MATERIALS

**Pull request merged**